sdk/freertos_app_cpu0/src/main.c

```diff
@@ -211,150 +135,7 @@ int main(void)
-                RxtaskCntr++;
-            }
-        }
-    }
-}
-
-/*-----------------------------------------------------------*/
-static void prvBlinkyTask(void *pvParameters)
-{
-    const TickType_t x250ms = pdMS_TO_TICKS(DELAY_1_SECOND / 4);
-    uint8_t led_offset = 0;
-
-    for (;;) {
-        if (message_status == 0) {
-            // If not complete, cycle yellow every 250ms
-            vTaskDelay(x250ms);
-
-            led_set_color(0 + led_offset, LED_COLOR_YELLOW);
-            led_set_color(1 + led_offset, LED_COLOR_BLACK);
-            led_set_color(2 + led_offset, LED_COLOR_BLACK);
-            led_set_color(3 + led_offset, LED_COLOR_BLACK);
-
-            led_offset = (led_offset + 1) % 4;
-        } else if (message_status == 1) {
-            // If complete, flash all green every 250ms
-            vTaskDelay(x250ms);
-
-            led_set_color(LED0, LED_COLOR_BLACK);
-            led_set_color(LED1, LED_COLOR_BLACK);
-            led_set_color(LED2, LED_COLOR_BLACK);
-            led_set_color(LED3, LED_COLOR_BLACK);
-
-            vTaskDelay(x250ms);
-
-            led_set_color(LED0, LED_COLOR_GREEN);
-            led_set_color(LED1, LED_COLOR_GREEN);
-            led_set_color(LED2, LED_COLOR_GREEN);
-            led_set_color(LED3, LED_COLOR_GREEN);
-        } else {
-            // message_status must be 2, meaning failure
-            // flash all red every 250ms
-            vTaskDelay(x250ms);
-
-            led_set_color(LED0, LED_COLOR_BLACK);
-            led_set_color(LED1, LED_COLOR_BLACK);
-            led_set_color(LED2, LED_COLOR_BLACK);
-            led_set_color(LED3, LED_COLOR_BLACK);
-
-            vTaskDelay(x250ms);
-
-            led_set_color(LED0, LED_COLOR_RED);
-            led_set_color(LED1, LED_COLOR_RED);
-            led_set_color(LED2, LED_COLOR_RED);
-            led_set_color(LED3, LED_COLOR_RED);
-        }
-    }
-}
-
-/*-----------------------------------------------------------*/
-static void vTimerCallback(TimerHandle_t pxTimer)
-{
-    long lTimerId;
-    configASSERT(pxTimer);
-
-    lTimerId = (long) pvTimerGetTimerID(pxTimer);
-
-    if (lTimerId != TIMER_ID) {
-        xil_printf("CPU0 - FreeRTOS Hello World Example FAILED");
-    }
-
-    /* If the RxtaskCntr is updated every time the Rx task is called. The
-     Rx task is called every time the Tx task sends a message. The Tx task
-     sends a message every 1 second.
-     The timer expires after 10 seconds. We expect the RxtaskCntr to at least
-     have a value of 9 (TIMER_CHECK_THRESHOLD) when the timer expires. */
-    if (RxtaskCntr >= TIMER_CHECK_THRESHOLD) {
-        message_status = 1;
-        xil_printf("CPU0 - FreeRTOS Hello World Example PASSED\r\n");
-    } else {
-        message_status = 2;
-        xil_printf("CPU0 - FreeRTOS Hello World Example FAILED\r\n");
+    	xil_printf("IF YOU'RE READING THIS THEN A TERRIBLE ERROR HAS OCCURRED!\n");
```

Comment:
Ha nice. :)

I am happy to keep this, but let's give the user a little more help here. Revise to say something about memory

Reply:
I need the deleted code to be kept as well, see above comment on deleted code. I think you can probably add a task for your own testing without deleting my tasks.